projects/hslayers/src/assets/locales/en.json

```diff
@@ -376,7 +376,9 @@
     "saveDrawingToLayer": "Save drawing to layer",
     "Select layer": "Select layer",
     "serverLayers": "Server layers",
-    "unsavedDrawing": "Unsaved drawing"
+    "unsavedDrawing": "Unsaved drawing",
+    "allowSnapping": "Allow snapping",
+    "snapTo": "Snap to features of layer:"
```

Comment:
I would leave the colon out of the translation string. For translating convenience.

Reply:
e7301c4